Framework/DataHandling/src/LoadILLReflectometry.cpp

```diff
@@ -793,13 +782,17 @@ void LoadILLReflectometry::initPixelWidth() {
 /// Update detector position according to data file
 void LoadILLReflectometry::placeDetector() {
   g_log.debug("Move the detector bank \n");
+  // if (m_localWorkspace->run().hasProperty("Distance.D1")) // Valid from 2018.
+  //  m_detectorDistance = inMeter(doubleFromRun("Distance.D1"));
+  // else // Valid before 2018.
+  //  m_detectorDistance = sampleDetectorDistance();
```

Comment:
Is this meant to be commented out? It should be removed or re-instated.

Reply:
Next line calls the `sampleDetectorDistance`. In the future, the Nexus file should contain a correct `D1` entry instead. Then, the method can be deleted